src/main/java/frc/robot/turret/TurretSubsystem.java

```diff
@@ -34,13 +43,16 @@ public Rotation2d getVelocity() {
         return inputs.velocity;
     }
 
+    public void setPower(double power) {
+        turret.setPower(power);
+    }
 
     public void handleRotateToPoint(Translation2d targetPoint, Translation2d robotPosition) {
         Translation2d normalizedRobotPosition = targetPoint.minus(robotPosition);
         this.turret.setPosition(
                 new Rotation2d(
-                        targetPoint.minus(robotPosition).getX(),
-                        targetPoint.minus(robotPosition).getY()
+                        normalizedRobotPosition.getX(),
+                        normalizedRobotPosition.getY()
                 )
         );
     }
```

Comment:
I think this function should only set control to target angle and if you want to do calcs on target angle maybe in another function or before u get it as param